src/database/models/team_item.rs

```diff
@@ -36,18 +39,21 @@ impl TeamBuilder {
                 user_id: member.user_id,
                 name: member.name,
                 role: member.role,
+                permissions: member.permissions,
+                accepted: member.accepted,
             };
 
             sqlx::query!(
                 "
-                INSERT INTO team_members (id, team_id, user_id, member_name, role)
-                VALUES ($1, $2, $3, $4, $5)
+                INSERT INTO team_members (id, team_id, user_id, member_name, role, permissions, accepted)
+                VALUES ($1, $2, $3, $4, $5, $6, TRUE)
```

Comment:
This should not be hardcoded to be `TRUE`.  Each team member has an individual accepted field, and we should be able to create teams with an initial owner (`accepted = true`) as well as initial team member requests (`accepted = false`).